### PR TITLE
Update the gemspec and reduce the published gems size

### PR DESCRIPTION
**Pull request by stevendaniels**
+ Removed the spreadsheets in ``test/files`` directory. Gem was reduced from 1.4MB to 63K!
+ Updated authors and email.
+ Removed the executables attribute--it's optional and unnecessary since roo doesn't have any executables.
+ Removed the ``test_files`` spec attribute, which is neither required or optional according to current [RubyGem Specs](http://guides.rubygems.org/specification-reference/).
+ Finally, it doesn't use open-ended dependencies.

### CODE REVIEW

File: roo.gemspec [hunk 1]
```diff
@@ -6,21 +6,20 @@ require 'roo/version'
...
 
   spec.files         = `git ls-files -z`.split("\x0")
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.files.reject! { |fn| fn.include?('test/files') }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'nokogiri'
-  spec.add_dependency 'rubyzip', '~> 1'
+  spec.add_dependency 'nokogiri', '~> 1.5'
```
Comment (simonoff) [inline roo.gemspec]: Why you limiting nokogiri to such version?

File: roo.gemspec [hunk 2]
```diff
@@ -6,21 +6,20 @@ require 'roo/version'
...
+  spec.add_dependency 'rubyzip', '~> 1.1', '>= 1.1.7'
```
Comment (simonoff) [inline roo.gemspec]: Why not leave `>= 1.1`, '< 2.0.0'

File: roo.gemspec [hunk 3]
```diff
@@ -6,21 +6,20 @@ require 'roo/version'
...
+  spec.email         = ['ruby.ruby.ruby.roo@gmail.com', 'oleksandr@simonov.me']
   spec.summary       = 'Roo can access the contents of various spreadsheet files.'
   spec.description   = "Roo can access the contents of various spreadsheet files. It can handle\n* OpenOffice\n* Excelx\n* LibreOffice\n* CSV"
   spec.homepage      = 'http://github.com/roo-rb/roo'
   spec.license       = 'MIT'
 
   spec.files         = `git ls-files -z`.split("\x0")
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.files.reject! { |fn| fn.include?('test/files') }
```
Comment (simonoff) [inline roo.gemspec]: what about `spec` ?

Comment (stevendaniels) [inline roo.gemspec]: Specs and tests are included in the `git ls-files -z` command

    $ ruby -e 'p `git ls-files -z`.split("\x0").select { |f| f.include? "spec.rb" }.size'
    # => 8

**Comment by stevendaniels** [comment]
I've updated the pull request to make the dependency range more forgiving.

**Pull request merged**